src/models/category.js

```
@@ -0,0 +1,38 @@
+  },
+  appearance: {
+    icon: {
+      type: String,
+      required: [true, FIELD_CANNOT_BE_EMPTY('icon')],
+      default: DEFAULT_CATEGORY_ICON
+    },
+    color: {
+      type: String,
+      required: [true, FIELD_CANNOT_BE_EMPTY('color')],
+      regex: HEX_COLOR_CODE_PATTERN,
+      default: DEFAULT_CATEGORY_ICON_COLOR
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)